formatTest/unit_tests/expected_output/infix.re

```diff
@@ -1084,7 +1084,7 @@ let server = {
         )
     )
     >>= (
-      (body) =>
+      body =>
```

Comment:
Hmm forgot this case in the callback PR.
The `body =>` should dock on the same line as `>>= (`, shouldn't it?

Reply:
I'm not sure - printing these have bigger issues - like knowing when exactly we can omit the parens. We're fairly certain that determining when the () can be omitted is such a hard problem that it requires integration with the parser.